WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeListActivity.java

```diff
@@ -282,10 +289,16 @@ public void onEventMainThread(PublicizeEvents.ActionRequestChooseAccount event)
 
         closeWebViewFragment();
 
+        SiteModel site = mSiteStore.getSiteBySiteId(event.getSiteId());
```

Comment:
Is it at all possible that this site might not be in the store? If it is, do we want to dismiss this event, or do we still want to keep going? Because we had an issue like that for notifications and [we had to do a hack](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java#L439) by initializing a new `SiteModel` instance with that `siteId` so everything would work.

I know we are already handling `null`, I just wanted to make sure that's what we wanted to do.